detector3.py

```diff
@@ -65,6 +65,8 @@ def __init__(self, spectrometer):
 
         self.started = False
 
+        self._stop_graph = False
```

Comment:
Same issues from sp-18

Reply:
this is still a propery, and a private attribute used like a public one, why not use a public one instead?

Reply:
I will change it